File: Modul-6/C/PRAK605_2410817210001_MuhammadFadillah.c
#include <stdio.h>

int main() {
    int r;
    scanf("%d", &r);
    int MatriksA[r][r], MatriksB[r][r], MatriksAXB[r][r];
    printf("Matriks A\n");
    for (int i = 0; i < r; i++) {
        for (int j = 0; j < r; j++) {
            scanf("%d", &MatriksA[i][j]);
        }
    }
    printf("Matriks B\n");
    for (int i = 0; i < r; i++) {
        for (int j = 0; j < r; j++) {
            scanf("%d", &MatriksB[i][j]);
        }
    }
    for (int i = 0; i < r; i++) {
        for (int j = 0; j < r; j++) {
            MatriksAXB[i][j] = 0;
            for (int k = 0; k < r; k++) {
                MatriksAXB[i][j] += MatriksA[i][k] * MatriksB[k][j];
            }
        }
    }
    printf("Matriks AXB\n");
    for (int i = 0; i < r; i++) {
        for (int j = 0; j < r; j++) {
            printf("%d ", MatriksAXB[i][j]);
        }
        printf("\n");
    }
    return 0;
}

File: Modul-6/Python/PRAK605_2410817210001_MuhammadFadillah.py
r = int(input())
print('Matriks A')
MatriksA = [list(map(int, input().split())) for _ in range(r)]
print('Matriks B')
MatriksB = [list(map(int, input().split())) for _ in range(r)]
print('Matriks AXB')
MatriksAXB = [[sum(MatriksA[i][k] * MatriksB[k][j] for k in range(r)) for j in range(r)] for i in range(r)]
for baris in MatriksAXB:
    print(*baris)